### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by bestwo01** [QVPR/Patch-NetVLAD#74]
Hello. Could you please tell me which loss function did you use and which part of the code shows the loss function?

**Comment by Tobias-Fischer** [comment]
Triplet loss, see https://github.com/QVPR/Patch-NetVLAD/blob/5597df22296124296a3691567064774e27af2e29/train.py#L192
